cmd/commands/root.go

```diff
@@ -69,7 +69,7 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&CephClusterNamespace, "namespace", "n", "rook-ceph", "Kubernetes namespace where CephCluster is created")
 }
 
-func GetClientsets(ctx context.Context) *k8sutil.Clientsets {
+func GetClientsets(ctx context.Context, preValidationCheck bool) *k8sutil.Clientsets {
```

Comment:
I would vote to always run the pre-validation checks, and no need for this flag.
```suggestion
func GetClientsets(ctx context.Context) *k8sutil.Clientsets {
```